contrib/download_cmake.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Script to download newest version of cmake on linux (or mac)
+# saves you the trouble of compiling it if you don't have root
+set -e # stop on failure
+cd "$(dirname "$0")"/.. # run in top-level directory
```

Comment:
Why not? This isn't being built from source like every single other dependency, it's downloading a binary.